src/stories/components/Teaser/styles.scss

```diff
@@ -0,0 +1,56 @@
+@import '../../../scss/variables.scss';
+
+// @file
+// Teaser
+
+.teaser {
+    width: 100%;
+    border: solid lightgrey;
```

Comment:
The color from the global variable doesn't match the sketch, so I used lightgrey instead.